pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -119,9 +119,20 @@ func haveOverlap(a1, a2 []string) bool {
 
 func (pl *VolumeRestrictions) PreFilter(ctx context.Context, cycleState *framework.CycleState, pod *v1.Pod) (*framework.PreFilterResult, *framework.Status) {
 	if pl.enableReadWriteOncePod {
-		return nil, pl.isReadWriteOncePodAccessModeConflict(ctx, pod)
+		if status := pl.isReadWriteOncePodAccessModeConflict(ctx, pod); !status.IsSuccess() {
```

Comment:
This is conflict with https://github.com/kubernetes/kubernetes/pull/114051, which changed the logics a lot in PreFilter to support preemption, let's merge that one first. 
/hold